docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -46,22 +47,30 @@ The solutions are located in the `getting-started-reactive` and `getting-started
 Quarkus is reactive.
 If you look under the hood, you will find a reactive engine powering your Quarkus application.
 This engine is Eclipse Vert.x (https://vertx.io).
-Every IO interaction passes through the non-blocking and reactive Vert.x engine.
+Every network I/O passes through the non-blocking and reactive Vert.x engine.
```

Comment:
I would say that I/O is a collective noun. So "all network IO" instead of "every".

Reply:
Fixed, thanks!